src/app/dashboard/dashboard.component.scss

```diff
@@ -1,15 +1,21 @@
 @import 'global_variables';
 @import 'global_states';
 
+.dash-wrapper {
+  padding: 0.925rem 4.5% 5rem;
+
+}
+
 @mixin icon {
-  font-size: 2rem;
-  text-align: left;
-  width: 36px;
-  height: 36px;
+  font-size: 2.333rem;
+  width: 40px;
+  height: 40px;
+  line-height: 40px;
```

Comment:
Should these be represented in `em` so they scale on different screens?

Reply:
They should. I've em'ed them